### PR TITLE
Add dockerfile based on python:3

### PR DESCRIPTION
**Pull request by Herbrant**

### CODE REVIEW

File: .dockerignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+*.github
+*.github
```
Comment (TendTo) [inline .dockerignore:2]: Maybe one of these was .git?

File: Dockerfile [hunk 2]
```diff
@@ -1,23 +1,9 @@
...
-
-ARG TOKEN=none
-ARG WEBHOOK_ENABLED=false
-ARG WEB_URL=none
-ARG GROUPS=none
-
-ENV BOT_DIR    /app
-
-WORKDIR $BOT_DIR
+FROM python:3
```
Comment (TendTo) [inline Dockerfile:1]: Can you use python:3-slim?

File: Dockerfile [hunk 3]
```diff
@@ -1,23 +1,9 @@
...
 COPY requirements.txt .
-#Install requirements
-RUN pip3 install --upgrade pip &&\
-  pip3 install -r requirements.txt
+RUN pip3 install -r requirements.txt
 
 COPY . .
-#Setup settings.yaml
-RUN mv config/settings.yaml.dist config/settings.yaml &&\
-  python3 settings.py -t ${TOKEN} -w ${WEBHOOK_ENABLED} -u ${WEB_URL} -g ${GROUPS}
```
Comment (TendTo) [inline Dockerfile:20]: But how all the configuration will be handled?

**Review by TendTo — CHANGES_REQUESTED** [review]

File: .dockerignore (hunk 1 above)
Comment (Herbrant) [reply .dockerignore:2]: Yes. I will fix it

File: Dockerfile (hunk 3 above)
Comment (Herbrant) [reply Dockerfile:20]: Secrets mustn't be passed as ARGS. Furthermore images mustn't contain configuration files. We will mount the configuration file on run

File: Dockerfile (hunk 2 above)
Comment (Herbrant) [reply Dockerfile:1]: Nope, some libraries required some packages that are not included in python:3-slim

**Review by TendTo — APPROVED** [review]

**Pull request merged**